src/main/resources/dbmigration/mysql/V3__createTables.sql

```diff
@@ -0,0 +1,81 @@
+drop table user;
```

Comment:
Never a good idea to drop table as it removes the data as well. And as User is the central table here, all other table define on top of it, will lose data as well. Generally we do modifications by altering tables schemas